code/globalincs/pstypes.h

```diff
@@ -288,7 +288,7 @@ constexpr bool LoggingEnabled = false;
 // Disabling this functionality is dangerous, crazy values can run rampant unchecked, and the longer it's disabled
 // the more likely you are to have problems getting it working again.
 #if defined(NDEBUG)
-#	define Assert(expr) do { ASSUME(expr); } while (false)
+#	define Assert(expr) do { ASSUME(expr); (void)sizeof(expr); } while (false)
```

Comment:
These changes will mark the values used in the expression as "used" and prevent warnings in release builds. This should have no effects on the generated code since everything is discarded but some usages of the assertion macros will now no longer work and required some adjustments.